portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,30 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+    private ArrayList<String> commentList = new ArrayList<String>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Sergio Ortiz!</h1>");
+      
+    initializeComments();
+    String json = convertToJsonUsingGson(commentList);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  private void initializeComments() {
+    commentList.add("Awesome Website!");
+    commentList.add("Great job! Keep it up!");
+    commentList.add("Very interesting!!");
+  }
+
+  /**
+   * Converts a ServerStats instance into a JSON string using the Gson library. Note: We first added
+   * the Gson library dependency to pom.xml.
+   */
+  private String convertToJsonUsingGson(ArrayList comments) {
+    Gson gson = new Gson();
```

Comment:
depending on how many times you plan to use this method in the class, you could consider making this a member variable so that you don't have to create the `Gson` object each time this method is called

Reply:
Good idea! Let me do that